### PR TITLE
Add -openrc subpackage to all aports shipping runscripts

### PR DESCRIPTION
**Pull request by nmeum**
Probably want to backport https://github.com/alpinelinux/abuild/commit/54dbf5f383a7a4248517fb7ddcb3bbf7d22f0d8b first

CC: @kaniini, @awilfox

### CODE REVIEW

File: community/pulseaudio/APKBUILD [hunk 1]
```diff
@@ -12,7 +12,7 @@ makedepends="m4 automake libtool intltool bash
 	dbus-dev libcap-dev alsa-lib-dev bluez-dev speexdsp-dev avahi-dev libressl-dev eudev-dev
 	libsndfile-dev gtk+3.0-dev json-c-dev fftw-dev gconf-dev sbc-dev jack-dev"
 install=""
-subpackages="$pkgname-dev $pkgname-doc $pkgname-libs $pkgname-bluez
+subpackages="$pkgname-openrc $pkgname-dev $pkgname-doc $pkgname-libs $pkgname-bluez
```
Comment (awilfox) [inline community/pulseaudio/APKBUILD]: This specifically already has -system (which is discouraged upstream), which takes the OpenRC scripts.

File: main/openrc/APKBUILD [hunk 2]
```diff
@@ -2,14 +2,14 @@
...
-pkgrel=6
+pkgrel=7
 pkgdesc="OpenRC manages the services, startup and shutdown of a host"
 url="https://gitweb.gentoo.org/proj/openrc.git"
 arch="all"
 license='BSD-2'
 depends=""
 makedepends="bsd-compat-headers"
-subpackages="$pkgname-doc $pkgname-dev"
+subpackages="$pkgname-openrc $pkgname-doc $pkgname-dev"
```
Comment (awilfox) [inline main/openrc/APKBUILD]: This seems silly :stuck_out_tongue:

File: main/clamav/APKBUILD [hunk 3]
```diff
@@ -4,7 +4,7 @@
 # Maintainer: Carlo Landmeter <clandmeter@gmail.com>
 pkgname=clamav
 pkgver=0.99.4
-pkgrel=0
+pkgrel=1
```
Comment (awilfox) [inline main/clamav/APKBUILD]: Why the empty bump?  This already had split scripts enabled.  Rebuilding should be a no-op.

Comment (nmeum) [reply main/clamav/APKBUILD]: That's indeed an error, probably fucked up during a rebase here.

File: main/openrc/APKBUILD (hunk 2 above)
Comment (nmeum) [reply main/openrc/APKBUILD]: Indeed

File: community/pulseaudio/APKBUILD (hunk 1 above)
Comment (nmeum) [reply community/pulseaudio/APKBUILD]: Thanks, will fix it

**Comment by nmeum** [comment]
Adressed the issues found by @awilfox and rebased against master.

File: main/qemu/APKBUILD [hunk 4]
```diff
@@ -45,7 +45,7 @@ install="$pkgname.pre-install $pkgname.post-install"
 # strip fails on .img files
 # some tests does not run on our builders
 options="suid !strip !check"
-subpackages="$pkgname-doc $pkgname-lang $pkgname-guest-agent:guest"
+subpackages="$pkgname-openrc $pkgname-doc $pkgname-lang $pkgname-guest-agent:guest"
```
Comment (jirutka) [inline main/qemu/APKBUILD]: This is in conflict with package [qemu-openrc](https://pkgs.alpinelinux.org/package/edge/main/x86_64/qemu-openrc).

Comment (nmeum) [reply main/qemu/APKBUILD]: Thanks, fixed!

**Comment by nmeum** [comment]
This needs to be rebased again, but before I do that: Do we want to include this in the next release? Because if so I can just go ahead rebase and merge it. Otherwise I would just put this on hold for now and invoke the scripts I used to generate this at a later point in time and recreate the changes from scratch.

**Pull request closed without merging**